CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 * Added methods `OrgVdcNetwork.Update`, `OrgVdcNetwork.UpdateAsync`, and `OrgVdcNetwork.Rename` [#292](https://github.com/vmware/go-vcloud-director/pull/292)
 * Added methods `EdgeGateway.Update` and `EdgeGateway.UpdateAsync` [#292](https://github.com/vmware/go-vcloud-director/pull/292)
-
+* Increment vCD API version used from 29.0 to 31.0
+    * Add fields `AdminVdc.UniversalNetworkPoolReference, VM.Media`
```

Comment:
Probably some morning typo

Reply:
6th line is additional info for 5th line